lib/cn/cn_tree.c

```diff
@@ -1825,7 +1888,10 @@ cn_comp_cleanup(struct cn_compaction_work *w)
                 }
             }
         } else if (w->cw_outv) {
-            cn_mblocks_destroy(w->cw_mp, w->cw_outc, w->cw_outv, kcompact);
+            uint n_out = spill ? 1 : w->cw_outc;
```

Comment:
Does w->cw_outv contain anything valid for spills?

Reply:
ah, not anymore with the subspill holding these things. Let me fix this.

Reply:
Also, we need a way to cleanup all enqueued subspills off all leaves if we run into an error.